src/layouts/sponsors.tsx

```diff
@@ -3,14 +3,14 @@ import * as React from 'react';
 import {
   Header,
   Headline,
+  Footer,
   SponsorInfo,
 } from '../components';
 
-
 export class Sponsors extends React.Component {
   public render(): JSX.Element {
     return (
-      <div className={'pt5'}>
+      <>
```

Comment:
What does an empty html element mean if you don't mind me asking?

Reply:
They are the short syntax for [React fragments](https://reactjs.org/docs/fragments.html#short-syntax).  You can return multiple elements without needing to put an extra `<div>` in the DOM at the top-level in your return statement.

Reply:
Pretty cool!